pom.xml

```diff
@@ -65,6 +73,12 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>${guava.version}</version>
```

Comment:
since gRpc depends on guava, I think you don't have to list the exact version here, no?